assets/templates/partials/_table-component.html

```diff
@@ -29,6 +29,11 @@
       <td>Content 4</td>
       <td>Content 5</td>
     </tr>
+    <tr class="sync">
+      <td colspan="1">Enderec (dplrvetex)</td>
+      <td colspan="1">Lista nueva</td>
+      <td colspan="3"><span class="text-sync"><i class="ms-icon icon-clock"></i> Synchronizing...</span></td>
```

Comment:
Should not the clock be part of the style?

Reply:
Si es cierto podría agregar el svg como background si les parece

Reply:
Creo que el problema de este approach es que el cliente igualmente debe indicar en que columna lo quiere, por lo que debería agregar una clase a la celda, tal vez lo más simple sea dejarlo tal cual está.

Reply:
ok

Reply:
Tiene razón Andrés. Me parece que el zebra es quien ayuda al usuario a diferenciar una a una cada fila de la lista. El sincronizando no debería  cambiar el background en caso de que se posicione en una fila de color gris, todos los datos interiores estarán en verde y además se visualizará la animación. Probemos como queda para verificar visualmente si nos hace algun ruido o se puede mejorar la usabilidad de otro modo. Pero comencemos por volver a respetar la zebra normalmente.
Muy bueno Gus! y muy bueno Andrés ese comment!

PD: creo que hice cagadas queriendo contestar esto y cerré el pull request. Lo volví a abrir y contesté. Les pido disculpas, aún estoy tratando de meterme en tema con esta plataforma.

Reply:
Joya, creo (por ahí me equivoco) que si aplicamos ese approach, el cambio va a ser incluso más chico que el actual y va a ser fácil de aplicar.

Si me estoy equivocando, y no es fácil hacer los cambios para probar esto, asumo que podemos salir con lo que está ahora y luego podemos mejorarlo en un futuro.

> PD: creo que hice cagadas queriendo contestar esto y cerré el pull request. Lo volví a abrir y contesté. Les pido disculpas, aún estoy tratando de meterme en tema con esta plataforma.

Jeje, no te preocupes, eso de cerrar los PRs sin querer es algo que nos pasó a casi todos cuando empezamos a usar github, incluso creo que es una muestra de que vas bien encaminado, jeje. Gracias por el esfuerzo!

Reply:
si, coincido completamente, en el caso de que se pueda mejorar visualmente de alguna manera más adelante, no significa que esta aplicación no sea valedera e incluso muy positiva.
Pero bueno, como todo, siempre se puede encontrar alguna opción mejor más adelante y pensar lo posible para llevarla a cabo. En fin, el trabajo de todos los días jaja.
Felicitaciones a ambos!!!